frameworks/bitcoin-move/sources/light_client.move

```diff
@@ -201,6 +201,12 @@ module bitcoin_move::light_client{
 
         // create new utxo
         handle_new_utxo(tx, &mut output_seals);
+
+        // handle bitseed
+        if(data_import_config::is_ord_mode(data_import_mode)) {
+            bitseed::process(tx);
```

Comment:
@jolestar I put the entry for parsing the bitseed protocol into light_client.move. Do you think it’s okay?